scipioninstaller/installer.py

```diff
@@ -310,19 +312,54 @@ def main():
 
         launcher = createLauncher(scipionHome, conda, dry, scipionEnv, dev)
         if not dry:
-            print("\n\nScipion has been successfully installed!! Happy EM processing!!\n\n")
-            print("You can launch Scipion using the launcher at %s\n" % launcher )
+            header = " ☻  Scipion has been successfully installed!! Happy EM processing!! ☻ "
+            content = "You can launch Scipion using the launcher at: %s " % launcher
+            tableLength = len(content) + len(content) % 2
+            createMessageInstallation(header, [content], tableLength)
 
     except InstallationError as e:
-        print(str(e))
-        print("Installation cancelled.")
+        header = "☹ Installation cancelled ☹ "
+        content = []
+        content.append("Error: ")
+        errors = str(e).split("\n")
+        for error in errors:
+            content.append(error)
+        content.append(" ")
+        content.append("For more information about the installation errors that can appear when installing or ")
+        content.append("using Scipion go to: https://scipion-em.github.io/docs/docs/user/troubleshooting.html ")
+        tableLength = len(content[-2]) if len(content[-2]) > len(content[1]) else len(content[1])
```

Comment:
Hi. If I understand well the propose of `tableLenght`. I suggest to make it inside the `createMessageInstallation` using something like
```
tableLenght = max([len(c) for c in content])
tableLenght = tableLenght + tableLenght % 2
```